### DISCUSSION TITLE
Help: Last letter missing when tagging

### DISCUSSION THREAD

**Discussion by glaubertorres** [Marekkon5/onetagger#413] [Q&A]
Hi,

I just discovered OneTagger and I really like it, its different from every software I tried before, but I am having some issues that I dont know how to solve:

Basically, whem Im tagging mannually all seems to work fine, but when search for the file the last latter is always missing in (genre), (artist), and other fields

In auto tag the same problem occurs, but it gets worse, because the software is replacing accented letters with special characters and symbols and completly skiping  some tracks even if the software marks as checked(image below)

![image](https://github.com/user-attachments/assets/c00bec09-b2f5-43a7-898d-87d08de8104d)

![image](https://github.com/user-attachments/assets/de243910-b870-4e4a-bd19-996fe784d61a)

**Comment by Marekkon5** [comment]
Hello, could you try the latest Actions build if the problem persists? Thanks

**Reply by glaubertorres** [reply]
Hey, I just installed the latest Actions build (#633) and its the same behavior.

**Reply by glaubertorres** [reply]
Hey, yesterday I installed the last build (634). It worked fine rigth after the installation, i thought the issue with last letter was solved but today I tried to use the software again and now I keep getting this error: 
![image](https://github.com/user-attachments/assets/7f652dff-bf60-4cc5-8767-5d96305000fe)

Auto tag doesnt appear to be working and Manual tag and Quick Tag doesnt let me save anything.

**Reply by Marekkon5** [reply]
Could you please provide the log? Settings > Advanced > Open Data Folder > onetagger.log 

Thank you

**Reply by glaubertorres** [reply]
[onetagger.log](https://github.com/user-attachments/files/16913785/onetagger.log)

**Reply by Marekkon5** [reply]
The error is permission denied, aka OneTagger cannot access the file. Perhaps some antivirus / anti-ransomware protection? Or the file is on a protected external drive.

You can try copying the file to different folder, or checking if OneTagger isn't flagged by AV, however OneTagger cannot do anything by itself since it's the OS preventing it to do so.
